Documentation/markdown/SetupHelp.md

```diff
@@ -0,0 +1,58 @@
+# OpenInfraPlatform Installation 
+
+***
+## Steps before the Installation 
+
+Before you can proceed to the Installation of **OpenInfraPlatform**, there are several steps, which have to be done before Installation: 
+
+1. Installation of **Visual Studio 2017** - 
+2. Installation of **CMake** -
+3. Installation of **Git** - 
+4. Creation of **GitHub account**
+5. Clone the **OpenInfraPlatform** repository - All information you will find in the **GitProcess.md**
+6. Installation of **Qt** -
+7. Installation of **Boost** -
+
+***
+## Installation 
+
+### 1. CMake 
+
+1.1 Open your **CMake**.
+
+1.2. In the line **Where is the source code:** copy the path to your source (e.g. C:\dev\Open-Infra-Platform).
+
+1.3. In the line **Where to build the binaries:**  copy the path to other folder (e.g. C:\dev\vs_projects). **NOTE:** This folder can not be in the **Open-Infra-Platform** folder as well as in the **Program Files** and **Windows** folders.
+
+1.4. Put a tick in the both check boxes *Grouped* and *Advanced*.
+
+1.5. Click *Configure*. 
+
+1.6. Select the Generator:
+
+1.6.1. Select "Visual Studio 15 2017 Win64".
+
+1.6.2.
```

Comment:
Select "x64". Here a screen dump would be great.

Reply:
Included CMake_Installation_settings.png